test/driver.ts

```diff
@@ -131,9 +148,15 @@ export class Driver {
 
     newParticipant(): Participant { // consumes two addresses from accounts for each participant - ethereum address and an orbs address
         const RESERVED_ACCOUNTS = 2;
-        const v = new Participant(this.accounts[RESERVED_ACCOUNTS + this.participants.length*2], this.accounts[RESERVED_ACCOUNTS + this.participants.length*2+1], this);
+        const v = new Participant(
```

Comment:
We've stretched the Participant object too much as both a validator and a non validator user. What do you think about separating them into two classes (by inheritance or possibly delegation)?

Reply:
Agree in general, but will not add much (or at all) to readability, so will add a TODO and leave as is for now.